### DISCUSSION TITLE
CSS not working anymore

### DISCUSSION THREAD

**Discussion by sarcyyy** [community/community#39847] [New to GitHub] [answered]
Hey! Sorry for my bad english.
Since today, my CSS isn't working on all my uploaded pages, and i have no idea why, i didn't changed anything at all and 2-3 days aggo it was working just fine, is it a general issues or i have to change something?

Thanks a lot!

**Comment by GodstimeNwabue** [comment]
check if it is linked and it has the correct path

**Reply by sarcyyy** [reply]
I didn't changed anything and today's it works just fine, no idea why, thanks for the answer tho!

**Reply by GodstimeNwabue** [reply]
Okay, glad it working now

**Comment by Preetiraj3697** [comment, marked as answer]
You check with code in html <div style="border:1px solid red"><div> if this CSS show in dom. Then you link is not working. also you go to browser inspect then you show you CSS that problem is CSS clash in different file. 
Also you go this discuss all answer is available here. https://github.com/community/community/discussions/22495 😊

**Reply by sarcyyy** [reply]
Thanks a lot!
